er_aws_elasticache/app_interface_input.py

```diff
@@ -173,6 +173,15 @@ def check_parameter_group_family(self) -> Self:
             )
         return self
 
+    @model_validator(mode="after")
+    def set_terraform_variables(self) -> Self:
+        """Set default tags for terraform"""
+        if not self.default_tags:
+            return self
+        with contextlib.suppress(KeyError):
+            self.default_tags_tf = self.default_tags[0]["tags"]
```

Comment:
which key can be missing?

Reply:
`tags` could be missing.

Reply:
how about just do it without suppress?

```python
self.default_tags_tf = self.default_tags[0].get("tags")
```

Reply:
This would always set `default_tags_tf` to `None`, but it may have another default value (in the future) in the Pydantic model.

Reply:
conditional override?

```python
if tags := self.default_tags[0].get("tags"):
    self.default_tags_tf = tags
```

Reply:
Sounds good. Done in 47ee067